portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,7 +28,32 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello Aymar!");
+    /*These lines used to return a Hello Aymar message*/
+    //response.setContentType("text/html;");
+    //response.getWriter().println("Hello Aymar!");
+    ArrayList<String> msg = new ArrayList<String>();
+    msg.add("First message.");
+    msg.add("This is the second string.");
+    msg.add("Last string.");
+    String json = convertToJson(msg);
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+
+  /**
+   * Converts an ArrayList into a JSON string using manual String concatentation.
+   */
+  private String convertToJson(ArrayList<String> msg) {
+    String json = "{";
+    int size = msg.size();
+    for(int i =0; i< size; i++){
+        json += "\"msg"+(i+1)+"\": ";
+        json += "\""+msg.get(i)+"\"";
+        if(i!=size-1){
+            json += ", ";
+        }
+    }
+    json += "}";
+    return json;
```

Comment:
Prefer using libraries to do the heavy lifting for you. We usually try to avoid implementing things on our own unless we really have to. That way we make our code less prone to bugs. (Assuming that the library you are using is well tested.)

Reply:
Fixed this